apps/src/labs/README.md

```diff
@@ -0,0 +1,21 @@
+# Lab2
+
+This folder contains the code for Lab2, which is our set of shared components and code for student labs.
+Lab2 is the preferred set of components for a new lab, as opposed to our legacy system (which includes StudioApp.js
+and project.js).
+
+## Guidelines for Lab2
+- Any code or component in `/labs` should be generic enough to be shared between multiple labs.
+- Code should be written in typescript.
+- Code should use functional React components.
+- Loading a level should only require api calls, not server data.
```

Comment:
I think the API calls in question do retrieve data from the server?

Reply:
fair, can you think of a way to write this that explains we don't want to rely on data sent over from the server on page load?

Reply:
Oh I see.  How about something like:

`When loading a lab as a ScriptLevel page, the lab part of the server-rendered page should not have any level- or user-specific data.  That information should be loaded afterwards using asynchronous calls to server APIs.  This allows the page to be cached independent of the user, and also allows the page to switch levels to other supported variants without a reload."

Reply:
thanks that's better!